dkg/protocol.go

```diff
@@ -10,6 +11,54 @@ type KeyGenOutput struct {
 	Share           *bls.SecretKey
 	OperatorPubKeys map[types.OperatorID]*bls.PublicKey
 	ValidatorPK     types.ValidatorPK
+	Threshold       uint64
+}
+
+type keygenOutputJSON struct {
+	Share           []byte
+	OperatorPubKeys map[types.OperatorID][]byte
+	ValidatorPK     types.ValidatorPK
+	Threshold       uint64
+}
+
+func (k *KeyGenOutput) MarshalJSON() ([]byte, error) {
+	rep := keygenOutputJSON{
```

Comment:
why can't a simple json.marshal work here?

Reply:
there's issue with using json.marshal for bls.SecretKey, bls.PublicKey

Reply:
Removed it.